templates/certified/model-details.html

```diff
@@ -82,7 +82,12 @@ <h3>{{ release.name }}</h3>
             <section>
             {% endif %}
             {% if release.level == "Enabled" %}
-              <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system’s hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
+              <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system's hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
+              {% if vendor == "Xilinx" %}
+                <p class="u-no-margin--bottom" style="padding-top:1.4rem;">
```

Comment:
Are you sure you need the online style here. If you remove the u-no-margin class above it will work?

Reply:
I did this for better whitespace consistency. For a board that is a un-certified and non-xilinx there won't be a button, so the margin from <p> and the padding form the <h4> below make a fairly large gap. The u-no-margin could be added based on whether the vendor is xilinx but I didn't want to added more statements